components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +151,102 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
 
 Just add members to any of these default policies, as described later, to quickly get your basic users up and running.
 
-### Policies and Roles in Automate
+## Working with Projects
 
-<!-- I do not like saying "user interface" because technically that includes the command line, too. But I can live with it if you think that is the lesser of two evils. -->
-IAM v2 introduces partial policy management from within Chef Automate in your browser.
+IAM projects are collections of resources created in Chef Automate or ingested from external data providers, including ChefInfra and ChefInspec.
+Projects are used in a policy to reduce the scope of permissions of that policy to just the resources tagged with the given projects.
+Without a policy statement to allow an operation tagged with a given project, a user cannot perform the actions of that policy.
+Note that Chef Automate is currently limited to six projects while we continue to refine the user experience during this beta period.
+
+There are four steps to setting up projects:
+
+1. Define each project via the UI (or the command line if you prefer).
+   Start on the projects list page shown previously to create your projects.
+2. Create or modify IAM policy statements to restrict permissions to specific projects as needed.
+   *Every* statement must either indicate specific projects or specify the wildcard denoting all projects (`*`).
+   By default, any previously existing policies are automatically setup with that wildcard so they apply to all projects.
+3. Tag resources with projects (see [Tagging Resources]({{< relref "iam-v2-overview.md#tagging-resources" >}})).
+4. Select the projects to filter in the UI.
+   Once you create some projects, use the **global project filter** in the penultimate position of the top navigation bar to select one or more projects for viewing.
+   With no selection (the default), you are thereby saying you want *no* filtering, hence displaying resources for *all* projects for which you have permission.
+
+Note that out-of-the-box we provide a *project-owner* role so the global admin may delegate much of these responsibilities for project management to others, to alleviate some of the burden for the global admin.
+
+### Properties of a Project
+
+A v2.1 project consists of a list of **rules**, where each rule describes a group of node characteristics.
+
+Property   | Description
+-----------|------------
+Rules      | list of tagging rules
+
+A rule consists of a list of **conditions**, where each condition describes a single characteristic.
+It also contains a type where <<TODO describe what node and event mean>>
+
+Property   | Description
+-----------|------------
+Type       | one of (node, event)
+Conditions | list of node characteristics
+
+A condition consists of these properties:
+
+Property   | Description
+-----------|------------
+Attribute  | for **event** types: one of (CHEF_ORGANIZATION, CHEF_SERVER)
```

Comment:
i think the first column should use _Event Attribute_ and _Node Attribute_ and just list the options in the description